### ISSUE TITLE
Windows Extension Manager View

### ISSUE THREAD

**Issue by vaughank92** [adobe/brackets#10811]
OS: Windows 7
Issue Summary:  When opening Brackets Extension Manager, if Brackets is reduced in size, the Extension Manager appears off the right side, resulting in it being cut off from view.
Brackets: Release 1.2 build 1.2.0-15697

![extensionmanager](https://cloud.githubusercontent.com/assets/6353702/6911885/dabe164a-d734-11e4-81d6-59bc98719576.png)

It might be just my personal preference but centered in the window might be a better option.  Are there any suggestions on where I would look to adjust this?

**Comment by prafulVaishnav** [comment]
@vaughank92 Can you try [disabling all extensions](https://github.com/adobe/brackets/wiki/Troubleshooting#disabling-extensions-individually) to make sure it's not due to any buggy extension

**Comment by vaughank92** [comment]
@prafulVaishnav When I disable all extensions I am unable to open the extension manager.  I went through instead and removed the couple I had (extract for brackets, brackets tools, and the Hello World tutorial extension).  Reinstalling the first two produces no problems.  Redoing the Hello World tutorial extension produces no problems now, the only thing I can guess is that somewhere in me messing with the basic extension I caused and error.  Silly me not seeing it.
